src/e2e-tests/services/anvil/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM ghcr.io/foundry-rs/foundry
+ARG FOUNDRY_VERSION=nightly-471e4ac317858b3419faaee58ade30c0671021e0
+FROM ghcr.io/foundry-rs/foundry:${FOUNDRY_VERSION}
```

Comment:
Chose a version from https://github.com/foundry-rs/foundry/pkgs/container/foundry that was compatible

`forge create --json` in entrypoint.sh wasn't returning the output in an expected formatting, causing the container to stop